src/core/client/stream/tabs/Live/LiveChatContainer.tsx

```diff
@@ -133,7 +134,7 @@ const LiveChatContainer: FunctionComponent<Props> = ({
     conversationState,
     showConversation,
     hideConversation,
-  ] = useConversation(eventEmitter);
+  ] = useConversation(eventEmitter, setLocal);
```

Comment:
Instead of passing `setLocal` here which is initialized with a lot more than just conversation related data, do a `useLocal` inside of `useConversation` and return the `conversationRootCommentID`.